### DISCUSSION TITLE
Multiple Camera Synchronization

### DISCUSSION THREAD

**Discussion by SoppierBrute9** [bonsai-rx/bonsai#1522] [Q&A]
Is there a way to synchronize three cameras, the tutorial only shows how to synchronize two cameras, but I need to do three.

**Comment by bruno-f-cruz** [comment]
How do you want to synchronize them in software?
1. Take the latest available frame from each camera each time one of them produces an event? (`CombineLatest`)
2. Pair frames from each camera (`Zip`)
3. Use one camera as the "Master" and sample from the other 2 (`WithLatestFrom` or `Sample`)

**Reply by SoppierBrute9** [reply]
I am not sure which would work best until I try all of them, but probably the take latest frame from each or use one camera as the master.

**Comment by SoppierBrute9** [comment]
Either take the latest available frame from each or use one camera as the master
